site/components/pages/Infinity/Cards/cards.ts

```diff
@@ -10,60 +18,103 @@ export const cards = [
     description:
       "We're proud to offset 10+ years of carbon emissions from Olympus to become Climate Positive. From the Ohmies, with Love.",
     tonsRetired: 33333,
-    date: "14 MAY 2022",
+    date: "21 APR 2022",
   },
   {
     logo: polygonLogo,
     link: "https://www.klimadao.finance/retirements/0x087a7afb6975a2837453be685eb6272576c0bc06",
     description:
-      "We care deeply about our work, and in turn we care deeply about preserving it. This also applies more generally, and we wish to preserve the planet which has afforded us this freedom.",
-    tonsRetired: 65201,
-    date: "10 MAY 2022",
+      "Together with Polygon, the broader blockchain industry needs to form a united front to fund, support, and leverage technoilogy that helps heal the earth, rather than destroy it",
+    tonsRetired: 104794,
+    date: "21 JUN 2022",
```

Comment:
Why not add logoWidth and logoHeight to every card?

What is the reason that only five cards do have these?
Either add these values to every card or to none?
How do future developers know when to add this and when not?

Reply:
Some of the images are already the correct size and some are too big so you can manually set height and width if needed. I was having trouble wrangling the images without setting width and height manually on the Image component.